streaming/streaming.go

```diff
@@ -365,7 +367,13 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		inputFilePattern = inputFilePattern + "/"
 	}
 	fmt.Println("Reading files from datastream destination ", inputFilePattern)
-
+	dataflowSubnetwork := ""
+	if dataflowCfg.Network != "" && dataflowCfg.Subnetwork == "" {
```

Comment:
nit: Merge these 2 conditions as they both check that Network is set.
Also, this method is becoming pretty large, should consider breaking it into smaller chunks

Reply:
Done